tests/topotests/pim_vrf_fhr_rp/pim_vrf_fhr_rp.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python
```

Comment:
please rename this to `test_pim_vrf_fhr_rp.py` otherwise this test will not be run automatically